images/cpg_pipes/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Standard analysis-runner driver image with cpg_pipes. 
+FROM ubuntu:20.04
+
+# hadolint ignore=DL3008, DL3013, DL4006, SC2086
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        apt-transport-https \
+        bash \
+        bzip2 \
+        ca-certificates \
+        curl \
+        git \
+        gnupg \
+        jq \
+        wget \
+        python3 \
+        python3-pip \
+        zip && \
+    pip install --no-cache-dir cpg_pipes==0.3.1  && \
+    # Google Cloud SDK
+    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
+    apt-get update && apt-get install -y google-cloud-sdk && \
+    # skopeo
```

Comment:
Why is skopeo needed as part of the image?

Reply:
Oh, because I wanted to originally go `FROM FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.0` and that didn't work for me, I just copied the base image directly and this was there. 
https://github.com/populationgenomics/analysis-runner/blob/main/driver/Dockerfile.base
Can certainly remove it though?

Reply:
That should be possible from this image builder, just won't be possible to test build it locally with that.

Reply:
Simplified now :)